src/initializers/kafka/kafka.ts

```diff
@@ -42,10 +42,11 @@ export default class Kafka {
         'api.version.request': true,
         'socket.blocking.max.ms': 100,
         'log.connection.close': false,
+        'enable.auto.commit': false,
```

Comment:
Should this also be configurable? Is this a breaking change?

Reply:
@nikostoulas I guess this has to do with the implementation of `base-kafka-handler`
https://github.com/Workable/orka/blob/7beb107871ba3748bc770b90e0c3cb370f75f9f4/src/initializers/kafka/base-kafka-handler.ts#L35
That calls explicitly `commit`, I guess @alkiskal can give us some more detail.

Reply:
Yes, the whole implementation of the handler depends on this to be false.

Reply:
oops! my bad, I was making some experiments, and forgot to remove it before opening the PR 😞